interfaces/evse_manager.yaml

```
@@ -112,6 +112,14 @@ cmds:
     result:
       description: Returns true if the signal was used by the evse manager implementation
       type: boolean
+  set_plug_and_charge_configuration:
+    description: >-
+      Sets the configuration required for an ISO15118 to handle contract authorization.
```

```suggestion
      Sets the configuration required for ISO15118 to handle contract authorization.
```

or `an ISO15118 module` ?